CDCgov/prime-reportstream
# 6801 Support ORC 14 repeating field values in FHIR

This PR adds support for ORC-14 repetitions.  See changes list.  Note the Linux4Health library does not support repetitions for extensions, and using variables does not work for repetitions either.

Test Steps:
1. Run the integration tests for this branch and verify they pass.
2. Compare `prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR_to_HL7/sample_co_1_20220518-0014.hl7` with `prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR_to_HL7/sample_co_1_20220518-0014-expected-base.hl7` and verify the beeper number now shows up in ORC-14

## Changes
- Changed the location of callback numbers to the telecom property of a practitioner instead of an extension.  Telecom was not being used AT ALL in practitioner.  I also checked the mapping tables and I did not see any future use of it either.
- Updated the ORC and OBR mapping tables in SharePoint with the new data location for ORC-14 and OBR-17
- Removed the `callback-number` extension
- Other tweaks to the data conversions
- Added a debug mode to elements.

## Checklist

### Testing

### Process

## Linked Issues
- #6801

## To Be Done


## Specific Security-related subjects a reviewer should pay specific attention to
